src/components/index.ts

```diff
@@ -295,5 +295,9 @@ export type {TruncateProps} from './Truncate';
 export {UnstyledLink} from './UnstyledLink';
 export type {UnstyledLinkProps} from './UnstyledLink';
 
+export {MediaCard} from './MediaCard';
```

Comment:
Please keep this list of exports in alphabetical order. 

Also do you want to export your prop interfaces for these two components?

Reply:
Ahh good catch! This changed from VideoCard to MediaCard and I had Maria do a find and replace 😁